setup.py

```diff
@@ -28,16 +30,18 @@
     "umap-learn>=0.3.7",
     "seaborn>=0.9.0",
     "hyperopt>=0.1.2",
-    "numba>=0.37, <0.45.1",
+    "numba>=0.37",
```

Comment:
maybe we need to != 0.45.1 here. It would be nice to check tox installs 0.46 and make sure this one works for UMAP. Maybe we should pin down with a comment that umap fails with this version of numba.